app/uk/gov/hmrc/cataloguefrontend/healthindicators/HealthIndicatorsConnector.scala

```diff
@@ -32,14 +32,21 @@ class HealthIndicatorsConnector @Inject()(
   import HttpReads.Implicits._
   private val logger = Logger(getClass)
 
+  implicit val rrR: Reads[RepositoryRating] = RepositoryRating.reads
+
   private val healthIndicatorsBaseUrl: String = servicesConfig.baseUrl("health-indicators")
 
   def getHealthIndicators(repoName: String)(implicit hc: HeaderCarrier): Future[Option[RepositoryRating]] = {
-    implicit val rrR: Reads[RepositoryRating] = RepositoryRating.reads
     val url                                   = s"$healthIndicatorsBaseUrl/health-indicators/repositories/$repoName"
```

Comment:
Formatting is all out of whack now that the other variable has been removed.
This is one of the issues with the "align fields" style  ...